0.17/files/docker-entrypoint.sh

```diff
@@ -29,7 +29,8 @@ if [ ! -f "$CONFIG/map-settings.json" ]; then
   cp /opt/factorio/data/map-settings.example.json "$CONFIG/map-settings.json"
 fi
 
-if find -L "$SAVES" -iname \*.tmp.zip -mindepth 1 -print | grep -q .; then
+NRTMPSAVES=$( find -L "$SAVES" -iname \*.tmp.zip -mindepth 1 | wc -l )
+if [ $NRTMPSAVES -gt 0 ]; then
```

Comment:
Could you combine those two lines? I don't see the need to assign a variable here.